Pod/Demo/AppDelegate.swift

```diff
@@ -104,6 +58,127 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
       return true
   }
 
+  func addPages() {
+    let font = UIFont(name: "ArialRoundedMTBold", size: 42.0)!
+    let color = UIColor.whiteColor()
+    let paragraphStyle = NSMutableParagraphStyle()
+    paragraphStyle.alignment = NSTextAlignment.Center
+
+    let attributes = [NSFontAttributeName: font, NSForegroundColorAttributeName: color,
+      NSParagraphStyleAttributeName: paragraphStyle]
+
+    let model1 = ContentViewModel(
+      title: "Tutorial on how to make a profit",
+      text: nil,
+      image: nil)
+    model1.setTitleAttributes(attributes)
+    model1.setTextAttributes(attributes)
+
+    let model2 = ContentViewModel(
+      title: "Step I",
+      text: "Collect underpants\n💭",
+      image: nil)
+    model2.setTitleAttributes(attributes)
+    model2.setTextAttributes(attributes)
+
+    let model3 = ContentViewModel(
+      title: "Step II",
+      text: "🎅🎅🏻🎅🏼🎅🏽🎅🏾🎅🏿",
+      image: nil)
+    model3.setTitleAttributes(attributes)
+    model3.setTextAttributes(attributes)
+
+    let model4 = ContentViewModel(
+      title: "Step III",
+      text: "Profit\n💸",
+      image: nil)
+    model4.setTitleAttributes(attributes)
+    model4.setTextAttributes(attributes)
+
+    let model5 = ContentViewModel(
+      title: nil,
+      text: "Thanks for your time.",
+      image:UIImage(named: "hyper-logo"))
+    model5.setTitleAttributes(attributes)
+    model5.setTextAttributes(attributes)
+
+    let page1 = UIViewController(model: model1)
+    let page2 = UIViewController(model: model2)
+    let page3 = UIViewController(model: model3)
+    let page4 = UIViewController(model: model4)
+    let page5 = UIViewController(model: model5)
+
+    tutorialController.add([page1, page2, page3, page4, page5])
+  }
+
+  func addBackgroundViews() {
+    let cloudView1 = UIImageView(image: UIImage(named: "cloud1"))
+    let backViewModel1 = BackViewModel(view: cloudView1,
+      position: Position(left: 0.7, top: 0.7))
+
+    tutorialController.addBackViewModels([backViewModel1])
+  }
+
+  func addAnimations() {
+    let image1 = UIImageView(image: UIImage(named: "cloud1"))
```

Comment:
and maybe similar for tutorialcontroller. with FP you can separate routine work (the same code) very easily